src/main/java/org/jabref/gui/fieldeditors/FieldNameLabel.java

```diff
@@ -14,5 +18,39 @@ public FieldNameLabel(Field field) {
         setPadding(new Insets(4, 0, 0, 0));
         setAlignment(Pos.CENTER);
         setPrefHeight(Double.POSITIVE_INFINITY);
+        setTip(field);
+    }
+
+    public void setTip(Field field) {
+        Tooltip tip = new Tooltip();
+        if (field.isStandardField()) {
+            StandardField standardField = (StandardField) field;
+            switch (standardField) {
+                case AUTHOR:
+                    tip.setText(Localization.lang("The name(s) of the author(s), in the format described in the LaTeX book. Remember, all names are separated with the and keyword, and not commas."));
```

Comment:
I would not refer to an external document (e.g. "described in the LaTeX book"), which is not properly accessible through a link. In My opinion, more (and local) information is better than the need of looking it up somewhere else.

Reply:
I would set quotation marks as well, when some specific syntax is addressed:
E.g. change `are separated with the and keyword` to `are separated with the "and" keyword`

Reply:
> I would not refer to an external document (e.g. "described in the LaTeX book"), which is not properly accessible. In my opinion, more (and local) information is better than the need of looking it up somewhere else. I see self-contained information where one needs it as the key. (If more information gets added to the tooltips, maybe the information could be structured in sections, split with new lines.)

Yes, thank you for your comment. I have the exact same feeling for these tips, I think that the context for each tip could be improved. I read those official documents, but I don't know how to summarize it properly.

Reply:
> I would set quotation marks as well, when some specific syntax is addressed:
> E.g. change `are separated with the and keyword` to `are separated with the "and" keyword`

This is better. Thanks!

Reply:
> Yes, thank you for your comment. I have the exact same feeling for these tips, I think that the context for each tip could be improved. I read those official documents, but I don't know how to summarize it properly.

I agree, this is not an easy task. Let's collaboratively see, how they can be improved. :)

Reply:
Can't we use the texts from our help page? --> https://docs.jabref.org/fields#standard-bibtex-fields

In case these text could be improved, feel free to submit a pull request.

I would say, the text for `author` can be improved ^^.

Reply:
Oh, another nice resource for additional information! :)